Changelog Modification for core-client and core-rest-pipeline (#15334)
Azure/azure-sdk-for-js: sdk/core/core-rest-pipeline/CHANGELOG.md

# Release History

## 1.1.0-beta.2 (2021-05-20)

> - Fixed an issue to check for the correct body length of the string and set the `Content-Length` header correctly. [PR 15314](https://github.com/Azure/azure-sdk-for-js/pull/15314)

Reviewer: nit
```suggestion
- Fixed an issue to set the `Content-Length` header correctly when using multibyte characters. [PR 15314](https://github.com/Azure/azure-sdk-for-js/pull/15314)
```
